src/hb.hh

```diff
@@ -213,6 +218,12 @@
 #include <winapifamily.h>
 #endif
 
+#ifndef PRId32
```

Comment:
Do you think that it is worth adding some of the others (PRId16, PRIu16, PRId8, etc) to make a basic subset of the macros?

Reply:
Let's add them as the need arises.

Reply:
Sounds good. Thanks for doing this.

Reply:
Good to merge?

Reply:
Yes.